README.md

```diff
@@ -127,6 +127,122 @@ When generating configuration, the program parses structured data (JSON) found i
 
 Similarly, the router watches its _own_ `routerConfig` annotations to dynamically construct global Nginx configuration.
 
+## Configuration Guide
+
+### Environment variables
+
+Router configuration is driven almost entirely by annotations on the router's replication controller and the services of all routable applications-- those labeled with `routable=true`.
```

Comment:
That is in reference to a label; not an annotation.  If we want to start prefixing label keys, that's a broader discussion because we're not currently prefixing the `app` or `heritage` labels, for instance, although I suppose an argument can be made that those two labels are special.